api/src/main/java/at/helpch/chatchat/api/Channel.java

```diff
@@ -19,5 +19,5 @@ public interface Channel {
 
     Set<User> targets(@NotNull final User source);
 
-    boolean isUseableBy(@NotNull final ChatUser user);
+    boolean isUsableBy(@NotNull final ChatUser user);
```

Comment:
No need to change this as I already am changing this in an open PR. Also is this really only ever used in 2 places? I could've swore it was more.